packages/sit-onyx/src/components/OnyxAccordionItem/OnyxAccordionItem.vue

```diff
@@ -27,10 +27,15 @@ defineSlots<{
 const itemId = useId();
 const accordionContext = inject(ACCORDION_INJECTION_KEY);
 
+const emit = defineEmits<{
+  (event: "update:open", accordionItem: string, value: boolean): void; // Event declaration
```

Comment:
```suggestion
  /** 
  * Emitted when the open state is changed.
  */
  "update:open": [isOpen: boolean]
```

To align with other component, we should use the new syntax :) Also I don't think we need to emit the itemId since its an internal ID used by the accordion